spec/index.bs

```diff
@@ -16772,10 +16717,12 @@ This feature adds the following [=optional API surfaces=]:
 Allows the {{GPUTextureUsage/RENDER_ATTACHMENT}} usage on textures with format {{GPUTextureFormat/"rg11b10ufloat"}},
 and also allows textures of that format to be blended, multisampled, and resolved.
 
+Implicitly allows {{GPUTextureFormat/"rg11b10ufloat"}} as a destination format in {{GPUQueue/copyExternalImageToTexture()}}.
```

Comment:
Will want to get confirmation from someone at Mozilla that this is OK.
Note we already require support for `rgb10a2unorm` and I think `rg11b10ufloat` will also be useful to people. But it's a bit harder to manually encode than `rgb11a2unorm`.

Reply:
Reread the notes and I think it's pretty clear that Jim said this is fine. Going to go ahead and merge.